src/configuration.rs

```diff
@@ -0,0 +1,8 @@
+use clap::Parser;
+
+/// Compliance crate configuration
+#[derive(Parser, Debug)]
+pub struct Configuration {
+    #[clap(short, long)]
```

Comment:
current convention is to support env vars with an `MC_` prefix
```suggestion
    #[clap(short, long, env = "MC_IP_INFO_KEY")]
```